### ISSUE TITLE
Use of extra_configs

### ISSUE THREAD

**Issue by PierreDurrr** [KBlixt/Movie-Extra-Downloader#12]
How am i supposed to use extra_configs ? Just past a 1080.cfg in this folder and start the script ?
I've modified required phrases from this .cfg but it seems to do nothing else than using default

**Comment by KBlixt** [comment]
correct. yeah, I didn't leave a lot of documentation on how to use the configs... that field should make the script disregard any youtube video that doesn't have the exact phrasing in the video title. It should be able to take in multiple phrases with a '|' delimiter.

how did you modify the required_phrases field? And what happens that you don't expect to happen? 

I do have plans to revisit this project after my current project. Hopefully I'll be able to improve reliability and customizability. And a few other things like renaming etc,etc.

**Comment by PierreDurrr** [comment]
Here are my required phrases, from the 1080p_trailer.cfg

required_phrases = bande annonce vf, trailer vf, vost, vf, Bande-Annonce Officielle Non Censurée (VF)

I expect it to pick YouTube video matching with one of this phrases, but it keep exclude video matching.

Here is my folder structure, is it OK ? 
root@scripts:/scripts/Movie-Extra-Downloader# tree                              .
├── default_config.cfg
├── default_extra_configs
│   └── 1080p-trailer.cfg
├── directory.py
├── extra_config.py
├── extra_configs
│   └── 1080p-trailer.cfg
├── extra_finder.py
├── failed_movies
├── LICENSE
├── main.py
├── Movie-Extra-Downloader.py
├── old_code
│   ├── config-example.cfg
│   ├── Movie-Extra-Downloader.py
│   ├── old_code.py
│   ├── Stream.py
│   └── YoutubeVideo.py
├── __pycache__
│   ├── directory.cpython-37.pyc
│   ├── extra_config.cpython-37.pyc
│   ├── extra_finder.cpython-37.pyc
│   ├── main.cpython-37.pyc
│   ├── tools.cpython-37.pyc
│   └── url_finders.cpython-37.pyc
├── README.md
├── records
├── tmp
│   └── tmp_0
├── tools.py
└── url_finders.py

**Comment by KBlixt** [comment]
Yeah, that usage make sense... 😅
That's not how past me thought to implement that though...

It should be a quick fix though. I'll let you know when I get it changed later today. 👍

**Comment by PierreDurrr** [comment]
At least it's good to know i'm not crazy 😁
No need to rush

**Comment by KBlixt** [comment]
give it a shot now, Please let me know if it works better :)

**Comment by PierreDurrr** [comment]
Yes it is working as expected now, thank you !
2 more things :

1.  can i change this value to whatever i want ?
```
# name of the subfolder that this config puts downloaded videos in.
extra_type = trailers
```

2. youtube-dl is downloading really slooow, less than 100 kb/s, do you know what can cause this ?

**Comment by KBlixt** [comment]
No, I'm afraid not, this is one of the things I've got planed for the rewrite.

Yeah noticed it as well yesterday, thought it was just me, probably something on YouTube-dl or on Youtubes end that is limiting the speed, maybe there is a new flag you need to pass in order to enable higher bandwidth. My use of YouTube-dl is fairly standard in this script I think.

Will investigate if it persists when I come back to the script.

**Comment by PierreDurrr** [comment]
Doesn't matter, your script is almost perfect 👍

**Comment by KBlixt** [comment]
😅 Yeah, pretty far from. But thanks.

If you find any false positives, let me know 👍
